### PR TITLE
add Feature 1 - Display Restaurant Listings Page

### PR DESCRIPTION
**Pull request by lvalerij**
**Description**
I added restaurant listing page with 2 sample restaurants

**How it was done**
I added listing of restaurants through map function. For that I used const array to display restaurants (stored in /common/ListExample.jsx)

### CODE REVIEW

File: src/components/RestaurantList/RestaurantList.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { ListWrapper } from "./RestaurantListElements";
+import { LIST_EXAMPLE } from "../../common/ListExample.jsx";
+
+export const RestaurantList = () => {
+  return (
```
Comment (katamatata) [inline src/components/RestaurantList/RestaurantList.jsx:6]: If the function has no logic in return, we can skip using return word and curly brackets. Like this:

```
export const RestaurantList = () => (
   <ListWrapper>
    ...
   </ListWrapper>
);
```

Comment (lvalerij) [reply src/components/RestaurantList/RestaurantList.jsx:6]: I would paste fetching data in this component, so it won't be without logic. Probably in the same component will be logic with filtering

Comment (katamatata) [reply src/components/RestaurantList/RestaurantList.jsx:6]: +1

File: src/App.jsx [hunk 2]
```diff
@@ -10,6 +10,7 @@ const App = () => (
     <Switch>
       <Route path="/" exact>
         <HomepageImage />
+        <RestaurantList />;
```
Comment (helloanil) [inline src/App.jsx:13]: Usually, we keep one Component per route. So how about creating a parent component like `HomePage` and put these two inside that component?

**Comment by helloanil** [comment]
+1

**Pull request merged**